nodewatcher/modules/platforms/openwrt/cgm/network.py

```diff
@@ -789,6 +789,14 @@ def network(node, cfg):
                     # Override default mcast_rate to avoid broadcast traffic from
                     # stealing too much air time
                     wif.mcast_rate = 6000
+                elif vif.mode == '802.11s':
+                    wif.mode = 'mesh'
+                    # Use BSSID from Mesh as Mesh ID
+                    wif.mesh_id = vif.bssid
+                    # Use full wpad instead of wpad-mini because of 802.11s Mesh.
+                    cfg.packages.update([
+                    'wpad -wpad-mini'
```

Comment:
Please don't specify multiple packages in the same string. Use two separate entries for that, for example:
```python
cfg.packages.update(['-wpad-mini', 'wpad'])
```